jekyll/_cci2/triggers-overview.adoc

```diff
@@ -48,10 +48,9 @@ TIP: If your code is integrated via **GitHub OAuth or Bitbucket Cloud**, you can
 
 . Select **Run Pipeline** and you will see your new pipeline running in the web app.
 
-=== Troubleshooting: "No suitable user credential could be found for user"
-
-If you are seeing "No suitable user credential could be found for user" when triggering a pipeline from the CircleCI web app, navigate to menu:Project Settings[Pipelines > Add Pipeline > Authorize] and then attempt to run the pipeline again by selecting btn:[Trigger Pipeline].
+NOTE: You may need to Authorize the connection to your GitHub account to continue, if you haven't yet.
 
+// @Rosie, is it worth adding an image here?
```

Comment:
@rosieyohannan not sure about this. 
Screenshot here: https://circleci.atlassian.net/browse/LIFE-1123?focusedCommentId=376931

Reply:
I actually think this is fine without the screenshot